native/src/components/TtsContainer.tsx

```diff
@@ -60,9 +61,13 @@ const TtsContainer = ({ children }: TtsContainerProps): ReactElement => {
   const enabled = buildConfig().featureFlags.tts
 
   const initializeTts = useCallback(async (): Promise<void> => {
-    await Tts.getInitStatus().catch(async error =>
-      error.code === 'no_engine' ? Tts.requestInstallEngine() : undefined,
-    )
+    await Tts.getInitStatus()
+      .catch(async error => (error.code === 'no_engine' ? Tts.requestInstallEngine() : undefined))
+      .then(() => {
+        if (Platform.OS === 'ios') {
+          Tts.setIgnoreSilentSwitch('ignore')
+        }
+      })
```

Comment:
🔧 Can we switch `catch` and `then` such that `then` comes first?

Reply:
Hmm, I think the catch should be before the then, since the error catching is pretty specific, and if there is an error in initializing TTS, there is no need to add any settings to it. But it's not something I feel strongly about, if you do, I'll happily change it.

The platform check shouldn't be necessary (I didn't actually test it) but I find it clearer to say that something that only affects iOS is only executed on iOS.

Reply:
> Hmm, I think the catch should be before the then, since the error catching is pretty specific, and if there is an error in initializing TTS, there is no need to add any settings to it. But it's not something I feel strongly about, if you do, I'll happily change it.

Then please move it out of the then block and use a separate `await` if its not related. I know that there was a mix with async/await and then/catch before already.

> The platform check shouldn't be necessary (I didn't actually test it) but I find it clearer to say that something that only affects iOS is only executed on iOS.

I am not sure it's a good idea to basically write code/add additional logic for basically documentation purposes. Not really sure myself though, so fine for me.